releaseNotes/SpringCloudAzureFeatureManagement.md

```diff
@@ -6,6 +6,33 @@
 
 [Source code web ][source_code_web] | [Package (Maven) web][package_web] | [Samples web][samples_web] | [Product documentation][docs]
 
+## 1.2.0 - January 13, 2020
+
+* Feature Management config files now match the DotNet format.
+
+```yaml
+FeatureManagement:
+  Beta: true
+  DarkTheme:
+    EnabledFor:
+      -
+        Name: Random
+        Parameters:
+          chance: "50"
+  BetaAB:
+    EnabledFor:
+      -
+        Name: Random
+        Parameters:
+          chance: "50"
+      -
+        Name: ClientFilter
+        Parameters:
+          clientIp: 10.0.0.1
+```
+
+* The isEnabled method is now non-blocking Async and has been renamed to isEnabledAsync.
```

Comment:
The reason for this change is to allow async feature filter. Anything the user needs to know to write an async feature filter?